docs/feature-flagging/randomization-sdk/api-keys.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 1
+---
+
+# API Keys
+
+You will need to initialize the SDK with an API key. The SDK uses this key to authenticate requests to Eppo's API for retrieving your experiment variations. Do not share your API token with anyone outside your organization.
+
+### Navigate to **API keys** section of the **Admin** tab
```

Comment:
@megan-eppo @vpai I think you had mentioned something about showing a GIF here instead?

Reply:
Yep, we are going to meet Friday to up-level the api key page doc assets. Will include either be better screenshots or gif(s). Not a blocker for this though!